TASK-37051: fixed redirection for space with name contains "-"

GouadriaHanen opened the pull request:
fixed : chat- clicking on the space logo which its name contains "-"

nidhalnidhal committed:
TASK-26741:Resolved merge conflicts (#238)

GouadriaHanen committed:
fix-37051:fixed redirectionfor a space with name contains ' - '

GouadriaHanen committed:
fix redirection with name space that contains multiple spaces

ahamdi commented on application/src/main/webapp/vue-app/components/ExoChatContact.vue:
```diff
@@ -192,8 +192,8 @@ export default {
     getSpaceURI() {
-      const spaceId = this.name.toLowerCase().split(' ').join('_');
-      getSpaceByPrettyName(this.name).then((space) => {
+      const spaceId = this.name.toLowerCase().split(' ').join('_').split('-').join('');
+      getSpaceByPrettyName(spaceId).then((space) => {
```
No need to recalculate the prettyName, it is already a propos of thi scomponent.
```suggestion
      getSpaceByPrettyName(this.prettyName).then((space) => {
```

GouadriaHanen replied:
done !

GouadriaHanen committed:
using pretty name

GouadriaHanen committed:
Merge branch 'feature/maintenance' into fix-37051-2

GouadriaHanen committed:
Merge branch 'feature/maintenance' into fix-37051-2

ahamdi committed:
37051 : Add groupId as a props of Chat Contact component
This Fix is composed of
 - Adding groupId as propos of Chat Contact component
 - Getting chat notifications correctly
